src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
@scott-eddy What is your HIL configuration for testing sensor noise addition? Is it one of the standard QGC options? I'm getting the impression that FlightGear and Xplane9 behave differently with high levels of noise on diff_pressure. I've also noticed that the QGC code which calculates airspeed from differential pressure is different for the two simulators, and that's probably not right.